src/components/send/clickToSendComponent.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './send.css';
+import Send from '../send';
+import { fromRawLsk } from '../../utils/lsk';
+
+
+const ClickToSendComponent = props => (
```

Comment:
I gave it some thought and I couldn't think of another name that exactly expresses what the component does. I like the name